README.md

```diff
@@ -1,148 +1,109 @@
+    Increment,
 }
 
-.foo {
-	background-color: red;
+impl Model for AppData {
+    fn event(&mut self, _: &mut Context, event: &mut Event) {
+        if let Some(app_event) = event.message.downcast() {
+            match app_event {
+                AppEvent::Increment => {
+                    self.count += 1;
+                }
+            }
+        }
+    }
 }
-```
-```rust
-Application::new(window_description, |cx|{
 
-	cx.add_style("style.css");
+Application::new(WindowDescription::new(), |cx| {
+    AppData { count: 0 }.build(cx);
 
-	HStack::new(cx, |cx|{
-		Label::new(cx, "Hello");
-		Label::new(cx, "World").class("foo");
-	});
+    HStack::new(cx, |cx| {
+        Button::new(cx, |cx| cx.emit(AppEvent::Increment), |cx| Label::new(cx, "Increment"));
+        Label::new(cx, AppData::count).width(Pixels(50.0));
+    })
+    .child_space(Stretch(1.0))
+    .col_between(Pixels(50.0));
 }).run();
 ```
-## State
-State describes the data provided by the user which the GUI will modify. VIZIA is reactive, which means that changes to the state will update the views.
- 
-### Defining State
-State is defined as a struct/enum which implements the `Model` trait:
-```rust
-#[derive(Lens)]
-struct AppData {
-	count: i32,
-}
 
-impl Model for AppData {}
+## Features
+ - Multiplatform (Windows, Linux, MacOS, Web)
+ - Declarative API
+ - Reactive with an elm-like archtecture
+ - Flexible layout engine, powered by [morphorm](https://github.com/vizia/morphorm)
+ - CSS-like styling with hot reloading
+ - Style property animations
 
-```
-The `Lens` derive macro enables binding views to fields of some state.
+## Learning Vizia
+<!-- Add docs to this when ready -->
+<div align="center">
+  <h3>
+    <a href="https://book.vizia.dev/"> Getting Started </a>
+    <span> | </span>
+    <a href="https://book.vizia.dev/"> The Book </a>
+    <span> | </span>
+    <a href="https://demo.vizia.dev/"> Web Demo </a>
+  </h3>
+</div>
 
-### Binding to State
-To use the state we need to `build` it into the view tree and then `bind` to it:
-```rust
-Application::new(window_description, |cx|{
-	
-	// Build the state into the tree
-	AppData{
-		count: 0,
-	}.build(cx);
-
-	HStack::new(cx, |cx|{
-		// Bind to the state
-		Binding::new(cx, AppData::count, |cx, some_data|{
-			Label::new(cx, *count.get(cx));
-		});
-	});
-}).run();
-```
-Anything within the body of the binding view will be updated when the bound data changes.
 
-### Mutating State
-To keep the separation of data changes and updates to the view tree, mutations of the data are done through events:
-```rust
-pub enum AppEvent {
-	Increment,
-	Decrement,
-}
+## Example Projects
 
-impl Model for AppData {
-	fn event(&mut self, cx: &mut Context, event: &mut MyEvent) {
-		if let Some(app_event) = event.message.downcast() {
-			match app_event {
-				AppEvent::Increment => self.count += 1,	
-				AppEvent::Decrement => self.count -= 1,
-			}
-		}
-	}
-}
-```
+Example projects go here... with pictures and stuff...
```

Comment:
Do you want me to make arborio look pretty so you can put it here?

Reply:
I would definitely love to put arborio here but there's no rush if styling is something you're leaving till later.